components/automate-ui/src/app/entities/users/user.actions.ts

```diff
@@ -2,7 +2,6 @@ import { Action } from '@ngrx/store';
 import { HttpErrorResponse } from '@angular/common/http';
 
 import { User } from './user.model';
-import { SelfUser } from './userself.model';
```

Comment:
Removing all the userself code from the user ngrx store files. This code was moved to the newly created userself ngrx store files.